pymc/distributions/distribution.py

```diff
@@ -208,6 +208,7 @@ def __init__(self, *args, ndim_supp, **kwargs):
         self.ndim_supp = ndim_supp
         kwargs.setdefault("inline", True)
         super().__init__(*args, **kwargs)
+        self.kwargs["ndim_supp"] = self.ndim_supp
```

Comment:
Is there a reason this has to happen after `super().__init__` and not before?

Reply:
I don't remember